src/platform/web/ui/session/room/timeline/TextMessageView.js

```diff
@@ -36,18 +36,29 @@ export class TextMessageView extends BaseMessageView {
  * Map from part to function that outputs DOM for the part
  */
 const formatFunction = {
+    header: headerBlock => tag["h" + Math.min(6,headerBlock.level)]({}, renderParts(headerBlock.inlines)),
```

Comment:
Should the min be on the model, or is your reasoning that it's a HTML limitation to only allow 6 nested headers?

Reply:
Indeed, my assumption was that 6 is an HTML limitation. I think that there's no real reason to enforce the magic number 6 on our "format-independent" data (although we'd probably be hard-pressed to find a source format that does use more than 6 headers)